<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_DefinirActionsEtTests</class>
 <widget class="QDialog" name="f_DefinirActionsEtTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Définition des actions et des tests</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/Images/App/icons_simple.png</normaloff>:/Images/App/icons_simple.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="LE_Nom">
     <property name="placeholderText">
      <string>Nom de l'élément</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="LB_Texte">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="LB_Module">
       <property name="text">
        <string>Module</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="CB_Module"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LB_ARU">
       <property name="text">
        <string>Code d'arrêt d'urgence</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LE_ARU"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="Btn_Action">
           <property name="text">
            <string>Ajouter une action</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_SupprAct">
           <property name="text">
            <string>Suppr une action</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="LV_Action">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="Btn_Test">
           <property name="text">
            <string>Ajouter un test</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_SupprTst">
           <property name="text">
            <string>Suppr une action</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="LV_Test">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="Btn_Ok">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
